helm-chart/eoapi/test-k3s-unittest-values.yaml

```diff
@@ -3,6 +3,10 @@ testing: true
 ingress:
   enabled: true
   className: "traefik"
+postgrescluster:
+  enabled: true
+  # Needed with PGO <= 5.7.0 https://github.com/CrunchyData/postgres-operator/issues/3152
+  backupsEnabled: true
```

Comment:
I think we should upgrade to PGO >= 5.7.0 in the tests instead.

Reply:
Yes indeed but I wanted to avoid testing the new release and to that later